after/plugin/indentLine.vim

```diff
@@ -365,7 +365,7 @@ endfunction
 augroup indentLine
     autocmd!
     if g:indentLine_newVersion
-        autocmd BufRead,BufNewFile,ColorScheme,Syntax * call <SID>InitColor()
+        autocmd BufEnter,BufWinEnter,BufRead,BufNewFile,ColorScheme,Syntax * call <SID>InitColor()
         if exists("##WinNew")
```

Comment:
<SID>InitColor() will be called too often.

Reply:
In the split wins, with different file type。In order to update indent line display after switch between wins，this may be necessary。
older vim works fine without this patch, but not in my vim 8.1 1-497. 
This patch fix it.

Reply:
> 
> 
> InitColor() will be called too often.

In the split wins, with different file type。In order to update indent line display after switch between wins，this may be necessary。
older vim works fine without this patch, but not in my vim 8.1 1-497.
This patch fix it.